test/tests/functional/pbs_provisioning_enhancement.py

```diff
@@ -94,9 +94,6 @@ def setUp(self):
         # Remove all nodes
         self.server.manager(MGR_CMD_DELETE, NODE, None, "")
 
-        # Restart PBS
-        self.server.restart()
```

Comment:
Was this intentional?

Reply:
It was, I noticed this restart was not serving any purpose in this test